OpenSSL/test/test_ssl.py

```diff
@@ -2376,6 +2376,7 @@ def test_app_data(self):
         :py:obj:`Connection.get_app_data`.
         """
         conn = Connection(Context(TLSv1_METHOD), None)
+        self.assertIsNone(conn.get_app_data())
```

Comment:
I'm not sure, sorry - I just saw there was a build failure and looked elsewhere in the file for an example of asserting for values equal to None.